Classes/StenoSignal.sc

```diff
@@ -43,7 +49,17 @@ StenoSignal {
 
 	// get filter input
 	filterInput { |argNumChannels, offset = 0|
-		^this.quelleInput(argNumChannels, offset) * env
+		var sig;
```

Comment:
I don't see the difference to `quelleInput` - why a separate implementation for `filterInput` rather than the forwarding we had before?

Reply:
for drafting reasons...

Reply:
ah good! :)